runtime/src/main/java/io/quarkiverse/ironjacamar/runtime/endpoint/TransactionAwareMessageEndpoint.java

```diff
@@ -79,4 +80,18 @@ public void afterDelivery() {
     public void release() {
         // Do nothing
     }
+
+    /**
+     * Check the transaction is active
+     */
+    boolean isActive() {
+        int currentStatus = QuarkusTransaction.getStatus();
+        return currentStatus == Status.STATUS_ACTIVE ||
```

Comment:
This can be simplified to `currentStatus != Status.NO_TRANSACTION`, like in the original code?

Reply:
The next `if` tests if it's rollback-only, maybe we can reuse this transaction status

Reply:
I reuse the codes from narayana https://github.com/jbosstm/narayana/blob/main/ArjunaJTA/cdi/classes/com/arjuna/ats/jta/cdi/TransactionContext.java#L138-L149

IIRC, if the status can not be `NO_TRANSACTION`, `STATUS_COMMITTED` and `STATUS_ROLLBACK`.